src/Umbraco.Cms.ManagementApi/Controllers/Dictionary/UploadDictionaryController.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Xml;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Hosting;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.ManagementApi.ViewModels.Dictionary;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.Dictionary;
+
+public class UploadDictionaryController : DictionaryControllerBase
+{
+    private readonly ILocalizedTextService _localizedTextService;
+    private readonly IHostingEnvironment _hostingEnvironment;
+
+    public UploadDictionaryController(ILocalizedTextService localizedTextService, IHostingEnvironment hostingEnvironment)
+    {
+        _localizedTextService = localizedTextService;
+        _hostingEnvironment = hostingEnvironment;
+    }
+
+    [HttpPost("upload")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(IActionResult), StatusCodes.Status200OK)]
```

Comment:
The response type is still wrong... IActionResult doesn't mean anything in this context, you're returning a `DictionaryItemsImportViewModel`